packages/validator/src/services/attestationDuties.ts

```diff
@@ -46,6 +47,19 @@ export class AttestationDutiesService {
     chainHeadTracker.runOnNewHead(this.onNewHead);
   }
 
+  remove(signer: PubkeyHex): void {
+    this.dutiesByIndexByEpoch.forEach((attDutiesAtEpoch, epoch) => {
+      attDutiesAtEpoch.dutiesByIndex.forEach((attDutyAndProof, vIndex) => {
+        if (toHexString(attDutyAndProof.duty.pubkey) === signer) {
+          attDutiesAtEpoch.dutiesByIndex.delete(vIndex);
+          if (attDutiesAtEpoch.dutiesByIndex.size === 0) {
+            this.dutiesByIndexByEpoch.delete(epoch);
+          }
+        }
+      });
+    });
```

Comment:
While forEach is totally valid syntax, please use for of to be consistent with the rest of the validator package style

Reply:
Replaced with `for of`